ivadomed/config/config_new_loader.json

```diff
@@ -33,8 +33,7 @@
     "split_dataset": {
         "fname_split": null,
         "random_seed": 6,
-        "center_test": [],
-        "method": "per_patient",
+        "data_testing": {"data_type": "subject", "data_value":[]},
```

Comment:
So (my memory is failing again): the strategy is to dev the new loader with this config file. And once the new loader is all standing up on its feet --> we will do the conversion of the other config file to the new format, correct?

Reply:
Correct, this would happen after issue #596 is handled.